pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -1476,7 +1476,7 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
             await this.writeFile('/usr/local/bin/wsl-exec', WSL_EXEC, 0o755);
             await this.runInit();
             if (rdNetworking) {
-              await this.runWslProxy();
+              this.runWslProxy();
```

Comment:
Should add a comment on we're explicitly not waiting. (Or make it not use `execCommand` and use `this.spawn()` instead, perhaps).